resource/ui/ts/models/timeseries.ts

```diff
@@ -1,5 +1,6 @@
 // source: models/timeseries.ts
```

Comment:
So should this file be renamed to Metrics?  Or should the module be renamed to Timeseries?